.changeset/changelog-stylelint.js

```diff
@@ -5,9 +5,44 @@
 const { getInfo, getInfoFromPullRequest } = require('@changesets/get-github-info');
 
 /**
- * @type {import('@changesets/types').ChangelogFunctions}
+ * @type {Array<string | undefined>}
+ */
+const CHANGESET_SECTIONS = ['Deprecated', 'Removed', 'Changed', 'Added', 'Fixed'];
+
+const changesetSectionReg = new RegExp(`^(${CHANGESET_SECTIONS.join('|')}): `);
```

Comment:
[suggest] `- ` at the begin of a line seems necessary:

```suggestion
const changesetSectionReg = new RegExp(`^- (${CHANGESET_SECTIONS.join('|')}): `);
```